hikari/api/rest.py

```diff
@@ -70,10 +70,6 @@ async def acquire(self, client: RESTClient) -> str:
             prefix.
         """
 
-    @abc.abstractmethod
-    async def close(self) -> None:
```

Comment:
Just a note for why I'm removing this, since this is intended for use with the OAuth2 flow and the refresh tokens Discord provides either don't expire or have a large enough expire time that nobody's noticed yet there's no need for any implementations of this to do anything which would need to be "closed" (i.e. have a background task)